apps/sms/test/marionette/lib/views/conversation/accessors.js

```diff
@@ -110,6 +110,16 @@ ConversationAccessor.prototype = {
     this.actions.flick(conversationContainer, 50, 50, 50, 350).perform();
   },
 
+  fakeScrollUpTo: function(scrollTop) {
+    var conversationContainer = this.client.findElement(
+      SELECTORS.container
+    );
+
+    conversationContainer.scriptWith(function(container, scrollTop) {
+      container.scrollTop = scrollTop;
+    }, scrollTop);
```

Comment:
__issue:__ you should use ```[scrollTop]``` see [1] - currently what it does in reality is ```container.scrollTop = undefined;```` 

[1] http://mozilla-b2g.github.io/marionette-js-client/api-docs/classes/Marionette.Element.html#method_scriptWith